contracts/src/Gateway.sol

```diff
@@ -617,4 +621,31 @@ contract Gateway is IGateway, IInitializable {
         assets.assetHubCreateAssetFee = config.assetHubCreateAssetFee;
         assets.assetHubReserveTransferFee = config.assetHubReserveTransferFee;
     }
+
+    /// @inheritdoc IGateway
+    function transact(
+        ParaID destinationChain,
+        OriginKind originKind,
+        uint128 destinationFee,
+        Weight calldata weightAtMost,
+        bytes calldata call
+    ) external payable {
+        if (call.length == 0 || destinationFee == 0 || weightAtMost.refTime == 0 || weightAtMost.proofSize == 0) {
```

Comment:
`call.length` must be at least 2 bytes. One for moduleId, and one for callIndex.

Reply:
https://github.com/Snowfork/snowbridge/pull/1141/commits/f8f0f57b29aae65c48ae0fe4fb577fe8cd67ac68